overview.md

```diff
@@ -114,7 +114,7 @@ Argument                  | Description
 `sourceFolder`            | (Optional) Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. If you specify `sourceFolder` and then generate a code coverage report, MATLAB uses only the source code in the specified folder and its subfolders to generate the report. You can specify multiple folders using a colon-separated or semicolon-separated list.<br/>**Example:** `source`<br/>**Example:** `source/folderA; source/folderB`
 `selectByFolder`          | (Optional) Location of the folder used to select test suite elements, relative to the project root folder. To create a test suite, MATLAB uses only the tests in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or semicolon-separated list.<br/>**Example:** `test`<br/>**Example:** `test/folderA; test/folderB`
 `selectByTag`             | (Optional) Test tag used to select test suite elements. To create a test suite, MATLAB uses only the test elements with the specified tag.<br/>**Example:** `Unit`
-`testResultsPDF`          | (Optional) Path to write test results report in PDF format. Currently, this argument is not supported on macOS platforms.<br/>**Example:** `test-results/results.pdf`         
+`testResultsPDF`          | (Optional) Path to write test results report in PDF format. On macOS platforms, this input is supported in MATLAB R2020b and later.<br/>**Example:** `test-results/results.pdf`         
```

Comment:
input -> argument

Is there a reason to use input? The table header mentions "argument."

Reply:
Done.